### PR TITLE
Enhancment config update #22

### LINKED ISSUE #22: Configuration update
Minor changes to @dragonraij 's configuration setup:

1. Addition to `.gitignore` file
2. Prettier code formatter setup
    1. Install prettier as a developer dependency.
    2. Prevent code formatting collision between ESLinter and Prettier: install `eslint-config-prettier`
    3. Configure `.eslintrc.json`

### PR DESCRIPTION
**Pull request by TheCandiko**
Minor updates for configuration settings:

1. Prettier install and ESLinter setup
2. `.gitignore` file has been changed.

Closes #22

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,163 @@
...
+
+# nyc test coverage
+.nyc_output
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# Bower dependency directory (https://bower.io/)
+bower_components
+
```
Comment (rajpranz) [inline .gitignore:67]: If you want to add grunt and bower to the project we can put it in at some stage. i had not added it initially as though would be good to keep things simple at first

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,163 @@
...
+# Windows Installer files
+*.cab
+*.msi
+*.msm
+*.msp
+
+# Windows shortcuts
+*.lnk
+
```
Comment (rajpranz) [inline .gitignore:161]: Its good idea that you added the additional ignore files so we are covered in the major environments.

I was using cloud9 as i found it easier as I was on different machines at different times

File: .eslintrc.json [hunk 3]
```diff
@@ -1,3 +1,6 @@
 {
-    "extends": "airbnb-base"
-}
\ No newline at end of file
+  "extends": ["airbnb-base", "prettier"],
```
Comment (rajpranz) [inline .eslintrc.json:2]: Yea we had mentioned we will be using prettier, I forgot to add it.

**Review by rajpranz — APPROVED** [review]
Thanks for adding the additional ignore profiles and adding the prettier plugin.  👍

File: .gitignore (hunk 1 above)
Comment (TheCandiko) [reply .gitignore:67]: I have this .gitignore file from gitignore.io website, bower dependency probably came under Node.js choice. For a bundler, I'm more familiar with Webpack and I think it is more popular. Maybe we can use that one? Not so much knowledge though.
I agree with keeping things simple at first, we can discuss later. :slightly_smiling_face:

File: .gitignore (hunk 2 above)
Comment (TheCandiko) [reply .gitignore:161]: C9 is very good platform, I've used a lot when I was following online bootcamp.
Though, local environment is always my first choice :slightly_smiling_face:

**Pull request merged**